src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --secondarycolor: #00acdc;
+  --primarycolor: #000;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 50px;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 15px 0;
```

Comment:
You should play with padding values here. Compare it with figma and finally you will find the correct value. (It should definitely by greater than 15)